### PR TITLE
Fix for issue #1512: Support for using multiple locators

### PR DESCRIPTION
**Pull request by badari412**
Fix for the issue #1512

### CODE REVIEW

File: src/SeleniumLibrary/locators/elementfinder.py [hunk 1]
```diff
@@ -76,6 +77,20 @@ def find(self, locator, tag=None, first_only=True, required=True, parent=None):
             )
         if self._is_webelement(locator):
             return locator
+
+        multi_locator_pattern = '.* >> (identifier|id|name|xpath|dom|link|partial link|css|class|jquery|sizzle|tag|scLocator)(:|=).*'
+
+        if not isinstance(locator, list) and Matcher(multi_locator_pattern, regexp=True).match(locator):
```
Comment (aaltat) [inline src/SeleniumLibrary/locators/elementfinder.py]: I would assume that it splitting could go separate method, at least it would be easier to test.

Comment (badari412) [reply src/SeleniumLibrary/locators/elementfinder.py]: Sure, I will extract the code to check for multiple locators and splitting to a separate method.

**Comment by badari412** [comment]
@aaltat Any idea why my tests are failing and how I can fix them? I just did a checkout of **master** branch on my ubuntu box and the tests are failing there too.

**Comment by aaltat** [comment]
Unit test is failing, it tests documentation and it should be fixed. See in https://github.com/robotframework/SeleniumLibrary/pull/1636/checks?check_run_id=1091285524

**Comment by badari412** [comment]
> 
> 
> Unit test is failing, it tests documentation and it should be fixed. See in https://github.com/robotframework/SeleniumLibrary/pull/1636/checks?check_run_id=1091285524

Thank you, I pushed the modified files and it worked!!

**Comment by aaltat** [comment]
Thank you for the contribution, but after thinking for a while, I did decided to do multiple locators little bit differently. See #1640 for more details. The work of #1640 is based on your idea and I will credit you in the release notes. Thank you for the contribution.

**Pull request closed without merging**